.github/workflows/dart.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
 
       - uses: nttld/setup-ndk@dbacc5871a0fac6eef9a09d2ca86bc8bf79432c3
         with:
-          ndk-version: r25b
+          ndk-version: r27
```

Comment:
Nice!

Reply:
Didn't work / not available from the source. Only increasing to r26b now.

I'm surprised the tests pass now. Are they only looking for the toolchain binaries but not invoking them?

Reply:
<img width="708" alt="image" src="https://github.com/dart-lang/native/assets/3601775/c507b15c-805b-426f-9696-12708f82dc20">

This should be invoking the tools you could double check the dylibs locally to see that it's build for the right arch.

Reply:
Oh, it uses -nostartfiles and links with nothing, so it's okay will all the standard libraries not existing yet :)

Reply:
Please file an issue that we should add tests for that on this repo.